iot-hub/Samples/service/ServiceClientSample/ServiceClientSample.cs

```diff
@@ -50,13 +54,50 @@ public async Task RunSampleAsync(TimeSpan runningTime)
 
         }
 
+        private async Task ReceiveMessageFeedbacksAsync(CancellationToken token)
+        {
+            _logger.LogInformation("Starting to listen to feedback messages");
+
+            while (!token.IsCancellationRequested)
+            {
+                var feedbackReceiver = _serviceClient.GetFeedbackReceiver();
+
+                try
+                {
+                    FeedbackBatch feedbackMessages = await feedbackReceiver.ReceiveAsync();
+                    if (feedbackMessages != null)

```

Comment:
nit: we could use a sleep here, so that we're not continuously looking for message feedback and pushing the cpu usage up.